chat.go

```diff
@@ -331,6 +331,23 @@ func MsgOptionDisableMarkdown() MsgOption {
 	}
 }
 
+// this function combines multiple options into a single option.
+func MsgOptionCompose(options ...MsgOption) MsgOption {
+	return func(c *sendConfig) error {
+		for _, opt := range options { opt(c) }
```

Comment:
you'll want to return the first error from applying the options.

forgot the signature included an error or I would have put that in the example.